pkg/limits/frontend/http.go

```diff
@@ -61,3 +104,50 @@ func (f *Frontend) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		RejectedStreams: resp.RejectedStreams,
 	})
 }
+
+func (s *RingStreamUsageGatherer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case http.MethodGet:
```

Comment:
Can we use `.Method` to separate out the implementations for `GET` and `POST`? i.e. https://github.com/grafana/loki/blob/main/pkg/loki/modules.go#L519